tests/send/send.spec.ts

```diff
@@ -0,0 +1,277 @@
+import { SendSelectors } from '@tests/selectors/send.selectors';
+import { WalletSelectors } from '@tests/selectors/wallet.selectors';
+import { TEST_PASSWORD } from '@tests/utils/constants';
+
+import { StacksTxFeeEstimation } from '@shared/models/fees/stacks-fees.model';
+import { RouteUrls } from '@shared/route-urls';
+
+import { formatErrorWithSymbol } from '@app/common/error-formatters';
+import { FormErrorMessages } from '@app/common/error-messages';
+
+import { json } from '../utils/json';
+import { test } from '../utils/testfn';
+
+test.describe('send crypto asset', () => {
+  let testAddress: string;
+
+  test.beforeEach(async ({ page, extensionId }) => {
+    await page.route(/.*/, route => route.abort());
+    await page.route(/chrome-extension/, route => route.continue());
+    await page.route(/github/, route => route.fulfill(json({})));
+    await page.route(/stacks.co.+balances$/, route => {
+      const res: AddressBalanceResponse = {
+        stx: {
+          balance: '1000000000',
+          total_sent: '0',
+          total_received: '0',
+          total_fees_sent: '0',
+          total_miner_rewards_received: '0',
+          lock_tx_id: '',
+          locked: '0',
+          lock_height: 0,
+          burnchain_lock_height: 0,
+          burnchain_unlock_height: 0,
+        },
+        fungible_tokens: {},
+        non_fungible_tokens: {},
+      };
+      return route.fulfill(json(res));
+    });
+    await page.route(/stacks.co.+fees/, route => {
+      const res: StacksTxFeeEstimation = {
+        cost_scalar_change_by_byte: 0.005,
+        estimated_cost: {
+          read_count: 0,
+          read_length: 0,
+          runtime: 0,
+          write_count: 0,
+          write_length: 0,
+        },
+        estimated_cost_scalar: 10,
+        estimations: [
+          { fee_rate: 0.1, fee: 100 },
+          { fee_rate: 2.0, fee: 200 },
+          { fee_rate: 30, fee: 300 },
+        ],
+      };
+      return route.fulfill(json(res));
+    });
+    await page.route(/nonce/, route => {
+      const res: AddressNonces = {
+        last_executed_tx_nonce: 1,
+        last_mempool_tx_nonce: null,
+        possible_next_nonce: 2,
+        detected_missing_nonces: [],
+      };
+      return route.fulfill(json(res));
+    });
+
+    await page.goto(`chrome-extension://${extensionId}/index.html`);
+
+    await page.getByTestId(OnboardingSelectors.AnalyticsDenyBtn).click();
+    await page.waitForURL('**' + RouteUrls.Onboarding);
+
+    await page.getByTestId(OnboardingSelectors.SignUpBtn).click();
+    await page.waitForURL('**' + RouteUrls.BackUpSecretKey);
+
+    await page.getByTestId(OnboardingSelectors.BackUpSecretKeyBtn).click();
+    await page.waitForURL('**' + RouteUrls.SetPassword);
+
+    await page.getByTestId(OnboardingSelectors.NewPasswordInput).fill(TEST_PASSWORD);
+    await page.getByTestId(OnboardingSelectors.SetPasswordBtn).click();
+    await page.waitForURL('**' + RouteUrls.Home);
```

Comment:
In the old test set up, we have helper methods so we don't need to repeat the sign up process in every test. Can that be reused?

Reply:
Yep, I intended to come back to this and refactor a bit. 👍

Reply:
@kyranjamie it appears we will want to use test fixtures and POM to simplify this: https://playwright.dev/docs/test-fixtures and https://playwright.dev/docs/pom

Refactoring to use this pattern.